### PR TITLE
docs: Compatibility document

### PR DESCRIPTION
**Pull request by isidentical**

### CODE REVIEW

File: docsite/docs/Docs/compatibility.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## Picking the right Fal version
+
+If you are starting a new project, we'd recommend you to use the latest Fal
+version in order to gain access to all the features Fal is offering. In the
+case of adoption to an existing project, you can use the tables below for
+picking the right version that would work best for your environment
+
+#### Fal \<> DBT Matrix
+
+|     | Fal Version | Supported DBT Version        |
```
Comment (chamini2) [inline docsite/docs/Docs/compatibility.md:16]: use lower case for dbt and fal throughout the doc

**Review by chamini2 — CHANGES_REQUESTED** [review]

File: docsite/docs/Docs/compatibility.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### From `after` scripts to `post-hook`s
+
+With the [`0.4.0`](https://github.com/fal-ai/fal/releases/tag/v0.4.0) release,
+Fal will start showing deprecation warnings for `after` scripts when using
+`fal flow run`. The easiest way forward is migrating them to `post-hook`s, and
+in general the migration is seamless. But if any of the scripts you have is
+using `write_to_source` or `write_to_model` functions, we'd recommend promoting
+them to individual ["Python models"](/Guides/python-models-migration).
+
+Please note that `post-hook`s can not be individually selected/run, so you have
```
Comment (chamini2) [inline docsite/docs/Docs/compatibility.md:40]: `can not` -> `cannot`

File: docsite/docs/Docs/compatibility.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+to select the underlying model and run all the `post-hook`s bound to it every
+time. If you need to select and run specific scripts, the current workaround is
```
Comment (chamini2) [inline docsite/docs/Docs/compatibility.md:42]: I don't know if we want to give a hack around this or maybe try to address it and document it once we have a good solution.

File: docsite/docs/Docs/compatibility.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## Migration
+
+### From `after` scripts to `post-hook`s
+
+With the [`0.4.0`](https://github.com/fal-ai/fal/releases/tag/v0.4.0) release,
+fal will start showing deprecation warnings for `after` scripts when using
+`fal flow run`. The easiest way forward is migrating them to `post-hook`s, and
+in general the migration is seamless. But if any of the scripts you have is
+using `write_to_source` or `write_to_model` functions, we'd recommend promoting
+them to individual ["Python models"](/Guides/python-models-migration).
```
Comment (chamini2) [inline docsite/docs/Docs/compatibility.md:38]: great!

**Review by chamini2 — APPROVED** [review]

**Review by burkaygur — APPROVED** [review]

**Pull request merged**